app/views/devise/passwords/edit.html.slim

```diff
@@ -1,10 +1,14 @@
-- content_for :devise_wrap do
+- content_for :devise_form do
   = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f|
     = f.govuk_error_summary
     = f.hidden_field :reset_password_token
-    = f.govuk_fieldset legend: { text: 'Change your password', tag: :h1, size: 'l' } do
-      = f.govuk_password_field :password, autocomplete: "new-password", label: { text: "New password" }, width: 'one-half', hint: { text: "(#{@minimum_password_length} characters minimum)" }
-      = f.govuk_password_field :password_confirmation, autocomplete: "new-password", label: { text: "Confirm new password" }, width: 'one-half'
-    = f.govuk_submit "Change my password"
+    = f.govuk_fieldset legend: { text: 'Choose a new password', tag: :h1, size: 'l' } do
```

Comment:
The tag and size are now defaults so can be pruned out if you check some recent work by @ebrett